src/connectionconfig/connectionDialogWebviewController.ts

```diff
@@ -1138,14 +1138,16 @@ export class ConnectionDialogWebviewController extends ReactWebviewPanelControll
                         string[] | undefined
                     >(azureSubscriptionFilterConfigKey) !== undefined;
 
+            const startTime = Date.now();
```

Comment:
Maybe we should create a helper function to track time. It would return another function that, when called, automatically measures the elapsed time and sends the action event with the mentioned properties.

Reply:
I like this thought.  I'll make an issue to add some util stuff like that.